README.md

```diff
@@ -56,6 +63,50 @@ This option allows you to disable it if you want:
 
 ```cordova run android ios -- --livereload --ghostMode=false```
```

Comment:
I thought the consensus was to completely separate the 2 options (--livereload and --devicesync), so they can each be specified independently without automatically invoking the other.

And why is it named --ghostmode? I realize that's what BrowserSync calls it but it's an odd name for the feature.